src/components/04-pages/partners/organization/experiences/experiences.stories.tsx

```diff
@@ -17,9 +18,113 @@ const meta: Meta<typeof PageLayout> = {
 
 export default meta;
 
+const experiencesTableColumns = [
+  { field: 'firstName', headerName: 'First Name', width: 150 },
+  { field: 'lastName', headerName: 'Last Name', width: 150 },
+  { field: 'email', headerName: 'Email', width: 150 },
+  { field: 'campus', headerName: 'Campus', width: 150 },
+  { field: 'offering', headerName: 'Offering', width: 150 },
+  { field: 'status', headerName: 'Status', width: 150 },
+  {
+    field: 'url',
+    width: 150,
+    type: 'actions',
+    headerName: 'Action',
+    disableExport: true,
+    renderCell: (params: GridRenderCellParams) => (
+      <strong>
+        <Button
+          component={Link}
+          href={
+            'http://localhost:6006/?path=/story/components-data-table--default/organization/experiences/' +
+            params.row.id
+          }
+          sx={{
+            fontWeight: 700,
+          }}
+        >
+          View details
+        </Button>
+      </strong>
+    ),
+  },
+];
+
+const experiencesTableRows = [
```

Comment:
@mcortes19 This example data is in the other file already let's pull it in from over there.